src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,1016 @@
+            "If you are running stack_recipes on Windows, make sure that you have "
+            "an associated application with executing .sh files. If you don't "
+            "have any and you see a pop-up during 'zenml stack_recipe run', we "
+            "suggest to use the Git BASH: https://gitforwindows.org/"
+        )
+
+    try:
+        _ = git_stack_recipes_handler.get_stack_recipes(stack_recipe_name)[0]
+    except KeyError as e:
+        cli_utils.error(str(e))
+    else:
+        stack_recipe_dir = stack_recipes_dir / stack_recipe_name
+        local_stack_recipe = LocalStackRecipe(
+            stack_recipe_dir, stack_recipe_name
+        )
+
+        if not local_stack_recipe.is_present():
+            ctx.invoke(
+                pull,
+                stack_recipe_name=stack_recipe_name,
+                path=path,
+                force=force,
+            )
+
+        try:
+            # check if terraform is installed
+            tf_client = Terraform(str(local_stack_recipe.path))
+            try:
+                tf_client.check_installation()
+            except RuntimeError as e:
+                cli_utils.error(str(e))
+
+            # set terraform log level
+            tf_client.set_log_level(log_level=log_level)
+
+            # Telemetry
+            track_event(
+                AnalyticsEvent.RUN_STACK_RECIPE,
+                {"stack_recipe_name": stack_recipe_name},
+            )
+            # use the terraform client to apply recipe
+            stack_yaml_file = os.path.join(
+                local_stack_recipe.path,
+                tf_client.apply(),
+            )
+
+            logger.info(
+                "\nA stack configuration YAML file has been generated as "
+                f"part of the deployment of the {stack_recipe_name} recipe. "
+                f"Find it at {stack_yaml_file}."
+            )
+
+            if not no_import:
+                logger.info(
+                    "\nThe flag --no-import is not set. Proceeding "
+                    "to import a new ZenML stack from the created resources."
+                )
+                import_stack_name = (
+                    stack_name if stack_name else stack_recipe_name
+                )
+                cli_utils.declare(
+                    f"Importing a new stack with the name {import_stack_name}."
+                )
+
+                # import deployed resources as ZenML stack
+                ctx.invoke(
+                    import_stack,
+                    stack_name=stack_name,
+                    filename=stack_yaml_file,
+                    ignore_version_mismatch=True,
+                )
+
+                cli_utils.declare(
+                    "Please consider creating any secrets that your stack "
+                    "components like the metadata store might need. "
+                    "You can inspect the fields of a stack component by "
+                    "running a describe command on them."
```

Comment:
Are these secrets still a part of the terraform outputs? If yes it might be great to instruct the user how to find the secrets as well

Reply:
That would be really convenient, I agree. I'll add instructions on how to access them.